jlm/rvsdg/region.hpp

```diff
@@ -203,8 +203,43 @@ class region {
 	void
 	append_argument(jlm::rvsdg::argument * argument);
 
-	void
-	remove_argument(size_t index);
+  /**
+   * Removes an argument from the region given a arguments' index.
```

Comment:
an arguments

Reply:
I am not sure I follow. What do you mean?

Reply:
Removes an argument from the region given a arguments' index.
 Removes an argument from the region given an arguments' index.

Reply:
Fixed.